addon/components/bs4/bs-collapse.js

```diff
@@ -1,2 +1,5 @@
-export { default } from 'ember-bootstrap/components/base/bs-collapse';
+import Collapse from 'ember-bootstrap/components/base/bs-collapse';
 
+export default Collapse.extend({
+  classNameBindings: ['collapse', 'in:show', 'collapsing']
```

Comment:
@simonihmig Looking for an opinion here. I overrode the `classNameBindings` that were in the base, reusing the `in` method. For this kind of thing should we generalize the method name and have `classNameBindings` defined separately in the bs3- and bs4-specific components?

I had a surprise at first because we already have a `show` method that isn't designed to support a class name binding.